internal/cluster/validations/pull_secret_validation.go

```diff
@@ -35,6 +36,16 @@ func ParsePublicRegistries(publicRegistries map[string]bool, publicRegistriesLit
 	}
 }
 
+func ParseMirrorRegistries(publicRegistries map[string]bool, mirrorRegistries []mirrorregistries.RegistriesConf) {
+	for _, mirrorRegistry := range mirrorRegistries {
+		registry, err := ParseRegistry(mirrorRegistry.Location)
+		if err != nil {
+			continue
```

Comment:
Should we be swallowing such an error or should this be logged/reported somehow?

Reply:
Added a log! What do you think?